client/js/models/archive.js

```diff
@@ -31,6 +32,22 @@ module.exports = {
     update: (data, state) => {
       return data
     },
+    updateDownloaded: (downloaded, state) => {
+      const meter = state.download_meter || speedometer(3)
+      return {
+        download_meter: meter,
+        downloaded: (state.downloaded || 0) + downloaded,
+        download_speed: meter(downloaded)
+      }
+    },
+    updateUploaded: (uploaded, state) => {
+      const meter = state.upload_meter || speedometer(3)
+      return {
+        upload_meter: meter,
+        uploaded: (state.uploaded || 0) + uploaded,
+        upload_speed: meter(uploaded)
+      }
+    },
```

Comment:
A few things:
* Seems like a choo best-practice is to explicitly define each model's default properties in the model.state property, so that at a glance another dev can come along and see all of the model's state properties that will be acted upon vs. having to pick thru all of the reducers. 
* We seem to be using camelCasing for state properties, so let's go with that. So for this commit I'd love to see in the archive model.state, even if they aren't defined at the outset (obvi) the properties: `uploadMeter: null`, `uploadSpeed = 0` and instead of plain `uploaded`, an `uploadedTotal` property, just to make it explicit. Same with download.